src/hiero_sdk_python/consensus/topic_message.py

```
@@ -1,14 +1,12 @@
-from typing import Optional, List, Union, Dict
-
-from hiero_sdk_python.timestamp import Timestamp
+from typing import Optional, List, Union
 from hiero_sdk_python.hapi.mirror import consensus_service_pb2 as mirror_proto
-from hiero_sdk_python.transaction.transaction_id import TransactionId
+
+def _to_datetime(ts_proto) -> datetime:
+    """
+    Convert a protobuf Timestamp to a Python datetime (UTC).
+    """
+    return datetime.utcfromtimestamp(ts_proto.seconds + ts_proto.nanos / 1e9)
```

Ruff (0.15.1)

[warning] `datetime.datetime.utcfromtimestamp()` used

(DTZ004)